tests/test_xlayer_cs.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=no-self-use
+import typing as t
+
+import capellambse.model.common as c
+from capellambse import MelodyModel
+
+
+def test_physical_path_has_ordered_list_of_involved_items(model: MelodyModel):
+    path = model.pa.all_physical_paths.by_name("card1 - card2 connection")
+    cmp_0 = model.pa.all_components.by_name("Compute Card 1")
+    link_1 = model.pa.all_physical_links.by_name("Eth Cable 2")
+    assert cmp_0 == path.involved_items[0]
+    assert link_1 == path.involved_items[1]
+
+
+def test_physical_path_has_ordered_list_of_involved_links(model: MelodyModel):
+    path = model.pa.all_physical_paths.by_name("card1 - card2 connection")
+    link_0 = model.pa.all_physical_links.by_name("Eth Cable 2")
+    link_1 = model.pa.all_physical_links.by_name("Eth Cable 3")
+    assert link_0 == path.involved_links[0]
+    assert link_1 == path.involved_links[1]
+
+
+def test_physical_path_has_exchanges(model: MelodyModel):
+    exchange = model.pa.all_component_exchanges.by_name("C 6")
+    path = model.pa.all_physical_paths.by_name("card1 - card2 connection")
+    assert exchange in path.exchanges
```

Comment:
If there's only supposed to be this one exchange, we'd probably want to assert that `path.exchanges == [exchange]` or similar. That way we also ensure that we don't mistakenly find unrelated exchanges.

Same thought applies to the other tests following below.

Reply:
for now it is only this one but as the test model evolves there might be more and so we'll have this test failing